test/externalTests/prb-math.sh

```diff
@@ -1,116 +0,0 @@
-#!/usr/bin/env bash
-
-# ------------------------------------------------------------------------------
-# This file is part of solidity.
-#
-# solidity is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# solidity is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with solidity.  If not, see <http://www.gnu.org/licenses/>
-#
-# (c) 2022 solidity contributors.
-#------------------------------------------------------------------------------
-
-set -e
-
-source scripts/common.sh
-source scripts/externalTests/common.sh
-
-REPO_ROOT=$(realpath "$(dirname "$0")/../..")
-
-verify_input "$@"
-BINARY_TYPE="$1"
-BINARY_PATH="$(realpath "$2")"
-SELECTED_PRESETS="$3"
```

Comment:
Ah right. I can add a command-line options for that.

Reply:
Now you can do:
```
./test/externalTests/prb-math.py native $(which solc) ir-no-optimize legacy-no-optimize
# or
./test/externalTests/prb-math.py native $(which solc)
```

Note that this still not working when using `external_tests.py`. It only works running each external test directly. I believe this is sufficient for debugging.

Reply:
Yeah, good enough.

I mean, I think that the original `externalTests.sh` passed all arguments to tests so it would work with it and we should eventually add that to `external_tests.py` as well. But for this PR it will do.